tests/e2e/CMakeLists.txt

```diff
@@ -1,25 +1,32 @@
 # Set up C++ functions for e2e tests
 function(add_query_module target_name src)
-  add_library(${target_name} SHARED ${src})
-  SET_TARGET_PROPERTIES(${target_name} PROPERTIES PREFIX "")
-  target_include_directories(${target_name} PRIVATE ${CMAKE_SOURCE_DIR}/include)
+        add_library(${target_name} SHARED ${src})
+        SET_TARGET_PROPERTIES(${target_name} PROPERTIES PREFIX "")
+        target_include_directories(${target_name} PRIVATE ${CMAKE_SOURCE_DIR}/include)
 endfunction()
 
-
 function(copy_e2e_python_files TARGET_PREFIX FILE_NAME)
-add_custom_target(memgraph__e2e__${TARGET_PREFIX}__${FILE_NAME} ALL
-        COMMAND ${CMAKE_COMMAND} -E copy
+        add_custom_target(memgraph__e2e__${TARGET_PREFIX}__${FILE_NAME} ALL
+                COMMAND ${CMAKE_COMMAND} -E copy
                 ${CMAKE_CURRENT_SOURCE_DIR}/${FILE_NAME}
                 ${CMAKE_CURRENT_BINARY_DIR}/${FILE_NAME}
-        DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${FILE_NAME})
+                DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${FILE_NAME})
+endfunction()
+
+function(copy_e2e_python_files_from_parent_folder TARGET_PREFIX EXTRA_PATH FILE_NAME)
```

Comment:
That deserves an explanation: from build/tests/e2e/replication/my_new_test.py I want to have access to the memgraph.py.MemgraphInstanceRunner.

Structure of tests folder is:

memgraph/tests/e2e/memgraph.py this is what we want to access. Note that this file is not copied yet to the /build folder as it is currently only used in memgraph/tests/e2e/runner.py
memgraph/tests/replication/show_while_creating_invalid_state.py: this is where I want to access the MemgraphInstanceRunner present in memgraph.py.

I have considered:

Not copying memgraph.py and use it directly from memgraph/tests/e2e/. I am not such fan of directly using the "source".
Copying memgraph.py from memgraph/tests/e2e/ to memgraph/build/tests/e2e/. However my understanding is that, in Python, it is easy to import function from another file from the same directory but it becomes more tricky when the file is in a different folder.
Copying memgraph.py from memgraph/tests/e2e/ to memgraph/build/tests/e2e/replication.

Let me know if anyone else has a better idea, I am open for suggestions.